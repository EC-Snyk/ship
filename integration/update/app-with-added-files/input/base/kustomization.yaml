kind: ""
apiversion: ""
resources:
- redis-deployment.yaml
