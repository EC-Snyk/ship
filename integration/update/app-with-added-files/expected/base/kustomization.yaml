kind: ""
apiversion: ""
resources:
- redis-deployment.yaml
- redis2-deployment.yaml
